### PR TITLE
docs: adds streamlined quick start

### PR DESCRIPTION
**Pull request by Dan-Nolan**
# Pull Request Checklist

- [x] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [x] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [x] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [x] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [x] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR adds submodules, updates Node.js setup, and enhances the Embedded Accounts quickstart guide with new sections and content.

### Detailed summary
- Added submodules for recursive fetching
- Updated Node.js setup
- Enhanced Embedded Accounts quickstart guide with new sections and content

> The following files were skipped due to too many changes: `site/pages/getting-started/walkthrough.mdx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/pages/getting-started/walkthrough.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# Walkthrough
+
+It's time to learn how it all works! Let's take a tour of the [starter app repository](https://github.com/alchemyplatform/embedded-accounts-quickstart) to see how it integrates with embedded accounts.
+
+## 1. The config
+
+First, let's do our configuration in a single file called `config.ts`.
+
+```ts [config.ts]
+// [!include ~/../examples/embedded-accounts-quickstart/src/config.ts:create-accounts-config]
```
Comment (moldy530) [inline site/pages/getting-started/walkthrough.mdx:16]: does the create-accounts-config section render correctly? I can't remember if I had issues with region names that were kebab cased

**Review by moldy530 — DISMISSED** [review]
overall LGTM had one question about region names, but I think my issue was different

**Comment by moldy530** [comment]
ah wait
```
./how-it-works.md in /home/runner/work/aa-sdk/aa-sdk/site/pages/getting-started/run-locally.mdx
./run-the-app.md in /home/runner/work/aa-sdk/aa-sdk/site/pages/getting-started/introduction.mdx
./how-it-works.md in /home/runner/work/aa-sdk/aa-sdk/site/pages/getting-started/introduction.mdx
```

you need to drop the extension on those files so they get rendered correctly

File: site/pages/getting-started/walkthrough.mdx (hunk 1 above)
Comment (Dan-Nolan) [reply site/pages/getting-started/walkthrough.mdx:16]: Yup it works! I actually thought it was kebab casing for a minute, but I had to change the region comments in the aa-sdk repo. After that it worked fine ✅

**Comment by Dan-Nolan** [comment]
good call, fixed the link extensions!

**Comment by Dan-Nolan** [comment]
actually, scratch that, the relative linking is off by one... 🤔

**Comment by Dan-Nolan** [comment]
checked all the links, working well ✅

**Comment by moldy530** [comment]
you might need to rebase on top of vocs-docs

**Pull request merged**